test/checks/aria/allowed-attr.js

```diff
@@ -57,7 +57,7 @@ describe('aria-allowed-attr', function () {
 	});
 
 	it('should return true if there is no role', function () {
-		var node = document.createElement('div');
+		var node = document.createElement('picture');
```

Comment:
After following https://github.com/w3c/html-aria/issues/52, it looks like `div` has no corresponding role (where I assumed it was `group`). Is that related?